CorsixTH/CMakeLists.txt

```diff
@@ -75,143 +75,145 @@ IF(APPLE)
-    IF(APPLE)
-      TARGET_LINK_LIBRARIES(CorsixTH libz.dylib)
-    ENDIF()
+if(CORSIX_TH_USE_FFMPEG)
+  find_package(FFmpeg COMPONENTS AVFORMAT AVCODEC AVUTIL SWSCALE SWRESAMPLE REQUIRED)
+  if(FFMPEG_FOUND)
+    target_link_libraries(CorsixTH ${FFMPEG_LIBRARIES})
+    include_directories(${FFMPEG_INCLUDE_DIRS})
+    if(APPLE)
+      target_link_libraries(CorsixTH libz.dylib)
+    endif()
     message("  FFmpeg found")
-  ELSE(FFMPEG_FOUND)
+  else()
     message(FATAL_ERROR "Error: FFmpeg library not found, even though it was selected to be included")
-  ENDIF(FFMPEG_FOUND)
-ENDIF(CORSIX_TH_USE_FFMPEG)
-
-IF(CORSIX_TH_USE_LIBAV)
-  FIND_PACKAGE(LibAV COMPONENTS AVFORMAT AVCODEC AVRESAMPLE AVUTIL SWSCALE REQUIRED)
-  IF(LIBAV_FOUND)
-    TARGET_LINK_LIBRARIES(CorsixTH ${LIBAV_LIBRARIES})
-    INCLUDE_DIRECTORIES(${LIBAV_INCLUDE_DIRS})
-    IF(APPLE)
-      TARGET_LINK_LIBRARIES(CorsixTH libz.dylib)
-    ENDIF()
+  endif()
+endif()
+
+if(CORSIX_TH_USE_LIBAV)
+  find_package(LibAV COMPONENTS AVFORMAT AVCODEC AVRESAMPLE AVUTIL SWSCALE REQUIRED)
+  if(LIBAV_FOUND)
+    target_link_libraries(CorsixTH ${LIBAV_LIBRARIES})
+    include_directories(${LIBAV_INCLUDE_DIRS})
+    if(APPLE)
+      target_link_libraries(CorsixTH libz.dylib)
+    endif()
     message("  LibAV found")
-  ELSE(LIBAV_FOUND)
+  else()
     message(FATAL_ERROR "Error: LibAV library not found, even though it was selected to be included")
-  ENDIF(LIBAV_FOUND)
-ENDIF(CORSIX_TH_USE_LIBAV)
+  endif()
+endif()
 
 # Find Freetype2
-IF(CORSIX_TH_USE_FREETYPE2)
-  FIND_PACKAGE(Freetype REQUIRED)
-  IF(FREETYPE_FOUND)
-    TARGET_LINK_LIBRARIES(CorsixTH ${FREETYPE_LIBRARIES})
-    INCLUDE_DIRECTORIES(${FREETYPE_INCLUDE_DIRS})
-	IF(APPLE)
-	  TARGET_LINK_LIBRARIES(CorsixTH libz.dylib)
-      TARGET_LINK_LIBRARIES(CorsixTH libbz2.dylib)
-    ENDIF()
+if(CORSIX_TH_USE_FREETYPE2)
+  find_package(Freetype REQUIRED)
+  if(FREETYPE_FOUND)
+    target_link_libraries(CorsixTH ${FREETYPE_LIBRARIES})
+    include_directories(${FREETYPE_INCLUDE_DIRS})
+    if(APPLE)
+      target_link_libraries(CorsixTH libz.dylib)
+      target_link_libraries(CorsixTH libbz2.dylib)
+    endif()
     message("  FreeType2 found")
-  ELSE(FREETYPE_FOUND)
+  else()
     message(FATAL_ERROR "Error: FreeType2 library not found, even though it was selected to be used")
-  ENDIF(FREETYPE_FOUND)
-ENDIF(CORSIX_TH_USE_FREETYPE2)
+  endif()
+endif()
 
 # Find msinttypes for MSVC
-IF(MSVC AND NOT CORSIX_TH_HAS_INTTYPES_H)
-  FIND_PATH(MSINTTYPES_INCLUDE_DIRS "inttypes.h" NO_DEFAULT_PATH)
-  MESSAGE(STATUS "Adding include directory: ${MSINTTYPES_INCLUDE_DIRS}")
-  INCLUDE_DIRECTORIES(${MSINTTYPES_INCLUDE_DIRS})
-  SET(CORSIX_TH_HAS_INTTYPES_H 1)
-ENDIF(MSVC AND NOT CORSIX_TH_HAS_INTTYPES_H)
-
-IF(MSVC AND CORSIX_TH_USE_VLD)
-  FIND_PACKAGE(VLD REQUIRED)
-  IF(VLD_FOUND)
-    TARGET_LINK_LIBRARIES(CorsixTH ${VLD_LIBRARY})
-    INCLUDE_DIRECTORIES(CorsixTH ${VLD_INCLUDE_DIR})
+if(MSVC AND NOT CORSIX_TH_HAS_INTTYPES_H)
```

Comment:
Ditto on dropping this block as MSVC has this header natively in VS2013+.
In addition we should change the only usage of `inttypes.h` in `SpriteEncoder/scanner.cpp` to use `<cinttypes>` instead.

Reply:
Yes to both, but the second is out of scope for this PR.